apps/ex_nvr/test/ex_nvr/devices_test.exs

```diff
@@ -232,6 +232,26 @@ defmodule ExNVR.DevicesTest do
     end
   end
 
+  describe "delete/1" do
+    setup ctx do
+      %{device: device_fixture(%{settings: ctx.settings})}
+    end
+
+    test "delete device", %{device: device} do
+      Enum.each(1..10, fn _value -> recording_fixture(device) end)
+      Enum.each(1..5, fn _value -> run_fixture(device) end)
```

Comment:
if you don't provide a run to recording it'll create one. So the 10 recordings above already created 10 runs. and use for for this:
```elixir
for _idx <- 1..10, do: recordings_fixutre(device)

assert :ok = Devices.delete(device)

refute ExNVR.Repo.get(Device, device.id)
# assert ...